### ISSUE TITLE
Revise design of access note(s) so that they're more visible 

### ISSUE THREAD

**Issue by ccleeton** [pulibrary/pulfalight#1370]
### Summary (Product Owner)

Ref: #1415 

Example: https://findingaids.princeton.edu/catalog/MC247_c2144?onlineToggle=false

During user testing, when asked if items in this series are restricted many users scroll right past the Conditions Governing Access

How can we make this more apparent visually to users? This design is just for the component page. The collection page has a badge the links to the relevant tab, which isn't presenting as a usability problem but may be worth consulting for consistency.

Current location:

![Image](https://github.com/user-attachments/assets/09b02c7e-e94a-45b5-ae8b-5115519224ac)

The goal is to quickly convey if there are access restrictions or not.

## Acceptance Criteria

- [ ] There's an agreed upon mockup or paper prototype for a new more prominent display of the conditions governing access & use

## First Step

Brainstorm some ways to make this content more visible.

Keep in mind - Sometimes there are very long conditions governing access, such as the following:

Collection level - https://findingaids.princeton.edu/catalog/AC137#access
folder level - https://findingaids.princeton.edu/catalog/AC137_c0109

## Useful Notes

It is programmatically possible to tell if a resource is open or not (`access_ssi`)

There's a badge, but it's not working for this use case.

### Duke

![Image](https://github.com/user-attachments/assets/2d5f7549-cb4f-4301-92cb-c69a737982c4)

POs liked this.

**Comment by ccleeton** [comment]
Adding sudden justification as it causes users to get frustrated not seeing the access restrictions and then coming to view materials in person that they can not see.

**Comment by ccleeton** [comment]
Christa going to speak to the ASpace Op group to see how these are pulled into the finding aids

**Comment by sdellis** [comment]
This is a design ticket. Success criteria to close this ticket would be a design mock-up posted to the comments that all Product Owners agree on. 

One suggestion would be to place `<accessrestrict>` notes above all the other content and to coordinate the style/colors with the Access Restrictions badge that is on the component.

**Comment by sdellis** [comment]
![access-pulfalight](https://github.com/user-attachments/assets/9245fc4b-e0b9-4773-8867-1ab5e6c06b8d)
Area with red box needs more prominence.

**Comment by ccleeton** [comment]
Oh we did comment on this somewhere else. It looks good but I was wondering how the big box would look on mobile?

**Comment by faithc** [comment]
Yes, here: https://github.com/pulibrary/pulfalight/issues/1446#issuecomment-2867364925

**Comment by sdellis** [comment]
@faithc @ccleeton  Ah, I knew I posted it, but apparently to the implementation ticket, not to the design ticket. Doh! At any rate, since this is only a mockup, there's no code to deploy to staging for further review. That would be part of the implementation ticket.

It's a Bootstrap Modal, so it should be responsive and look good on a phone. See the image of a screengrab of an iPhone 11 simulation from their demo page (in dark mode). We can certainly test the implementation on staging before merging and deploying to production. 

<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUVCQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2834df1fb3759673070d708b88ec30fcdca1bef0/Screenshot%202025-05-13%20at%203.23.40%E2%80%AFPM.png" alt="Screenshot 2025-05-13 at 3.23.40 PM.png" />

**Comment by faithc** [comment]
I can't see the screenshot, but all of that sounds good to me. :)

**Comment by ccleeton** [comment]
Ditto!
